### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by rockdaboot**
Fix spelling reported by `codespell`.

### CODE REVIEW

**Review by pellared — APPROVED** [review]

File: internal/include/libbpf/bpf_helper_defs.h
Comment (MrAlias) [inline internal/include/libbpf/bpf_helper_defs.h:1]: These files in libbpf were copied from https://github.com/libbpf/libbpf. Updating spelling issues here has good potential to cause trash when we re-sync. Can we skip these?

@grcevski

Comment (MrAlias) [reply internal/include/libbpf/bpf_helper_defs.h:1]: Related: https://github.com/libbpf/libbpf/pull/819

Comment (rockdaboot) [reply internal/include/libbpf/bpf_helper_defs.h:1]: Updated and rebased

**Comment by rockdaboot** [comment]
Superseded by #863

**Pull request closed without merging**